src/main/java/uk/gov/justice/hmpps/prison/api/resource/CaseNoteResource.java

```diff
@@ -72,73 +59,61 @@ public List<CaseNoteStaffUsage> getCaseNoteStaffUsageSummaryByPost(@RequestBody
+    @PreAuthorize("hasRole('VIEW_CASE_NOTES')")
+    public List<CaseNoteUsage> getCaseNoteUsageSummary(
+        @RequestParam(value = "offenderNo", required = false) @Parameter(description = "a list of offender numbers to search.") final List<String> offenderNo,
+        @RequestParam(value = "staffId", required = false) @Parameter(description = "Staff Id to filter by") final Integer staffId,
+        @RequestParam(value = "agencyId", required = false) @Parameter(description = "Agency Id to filter by") final String agencyId,
+        @RequestParam(value = "numMonths", required = false, defaultValue = "1") @Parameter(description = "Number of month to look forward (if fromDate only defined), or back (if toDate only defined). Default is 1 month") final Integer numMonths,
+        @RequestParam(value = "fromDate", required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) @Parameter(description = "Only case notes occurring on or after this date (in YYYY-MM-DD format) will be considered.  If not defined then the numMonth before the current date, unless a toDate is defined when it will be numMonths before toDate") final LocalDate fromDate,
+        @RequestParam(value = "toDate", required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) @Parameter(description = "Only case notes occurring on or before this date (in YYYY-MM-DD format) will be considered. If not defined then the current date will be used, unless a fromDate is defined when it will be numMonths after fromDate") final LocalDate toDate,
+        @RequestParam(value = "type", required = false) @Parameter(description = "Case note type.") final String type,
+        @RequestParam(value = "subType", required = false) @Parameter(description = "Case note sub-type.") final String subType
+    ) {
         return caseNoteService.getCaseNoteUsage(type, subType, offenderNo, staffId, agencyId, fromDate, toDate, ObjectUtils.defaultIfNull(numMonths, 1));
     }
 
     @ApiResponses({
             @ApiResponse(responseCode = "200", description = "The case note usage list is returned.")})
-    @Operation(summary = "Retrieves list of case notes grouped by type and offender", description = "Retrieves list of case notes grouped by type and offender")
+    @Operation(summary = "Retrieves list of case notes grouped by type and offender", description = "Retrieves list of case notes grouped by type and offender. Requires role VIEW_CASE_NOTES")
     @PostMapping("/usage")
     @SlowReportQuery
+    @PreAuthorize("hasRole('VIEW_CASE_NOTES')")
```

Comment:
just checking that the clients calling this now have the VIEW_CASE_NOTES role (if they didn't already) possibly hmpps-prisoner-profile-system etc?